logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/wireuser/search/SearchPublicUserUseCase.kt

```diff
@@ -1,29 +1,29 @@
-package com.wire.kalium.logic.feature.user
+package com.wire.kalium.logic.feature.wireuser.search
 
 import com.wire.kalium.logic.CoreFailure
-import com.wire.kalium.logic.data.publicuser.PublicUserRepository
-import com.wire.kalium.logic.data.publicuser.model.PublicUserSearchResult
+import com.wire.kalium.logic.data.wireuser.WireUserRepository
 import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.functional.map
 
 
 interface SearchPublicUserUseCase {
```

Comment:
As `Public` got renamed to `Wire` shouldn't this change as well so it reflects the naming better?